tests_e2e/orchestrator/lib/agent_test_suite_combinator.py

```diff
@@ -202,11 +222,14 @@ def create_environment_list(self) -> List[Dict[str, Any]]:
                     # create an environment for exclusive use by this suite
                     environment_list.append({
                         "c_marketplace_image": marketplace_image,
+                        "c_cloud": self.runbook.cloud,
                         "c_location": location,
                         "c_vm_size": vm_size,
                         "c_vhd": vhd,
                         "c_test_suites": [suite_info],
-                        "c_env_name": f"{name}-{suite_info.name}"
+                        "c_env_name": f"{name}-{suite_info.name}",
+                        "c_marketplace_image_information_location": self._MARKETPLACE_IMAGE_INFORMATION_LOCATIONS[self.runbook.cloud],
```

Comment:
how these are different from c_location. will Lisa use these two to create vm in this location rather than test_suite/image location?

Reply:
they are unrelated to the location for the test VM. LISA keeps some caches and creates them on those regions.